services/cards/src/actions/Filters.tsx

```diff
@@ -12,13 +12,11 @@ const qs = require('qs');
 // Filter changes trigger several things, including the actual FiltersChange action
 export function filterChange(name: string, value: string | number): ((dispatch: Redux.Dispatch<any>) => void) {
   return (dispatch: Redux.Dispatch<any>) => {
-    if (name === 'source' && value === 'Custom') {
+    let cardstype = null;
+    if (name === 'source' && value === 'custom') {
       // TODO validate URL or ID, otherwise notify user + abort
-      value = window.prompt('Please enter your card sheet publish URL (see "?" in the top right for help)', '') as string;
-      if (value.indexOf('/e/') !== -1) {
-        return alert('Please use the URL of the Google Doc, not the publish link');
-      }
-      value = 'Custom:' + value.replace('https://docs.google.com/spreadsheets/d/', '').split('/')[0];
+      cardstype = window.prompt('Please enter a sheet type (Ability, Adventurer, Encounter, Helper, Loot, Persona, Skill)');
```

Comment:
1. Is it possible to get the tab name from the API so that we don't have to ask the user?
2. Whatever the final flow is (incl how users can generate the new URLs, if that's changed), can you update the docs too? https://github.com/ExpeditionRPG/expedition/blob/master/services/cards/CARD-CREATION.md

Reply:
1. The API doesn't include the sheet type anywhere - so we can't infer it without some fuzzy matching or modifying the structure of the sheets - both of which I'd prefer not to do in an "unbreak" PR
2. Done